props/saber_sa22c_buttons.h

```diff
@@ -0,0 +1,334 @@
+// sa22c props file, includes the following changes
+//
+// New #define NO_LOCKUP_HOLD
```

Comment:
Could you prefix this define with SA22C_ so that we won't run into any collisions?